### ISSUE TITLE
Module import causes errors in julia's shell mode

### ISSUE THREAD

**Issue by phobon** [stevengj/PyCall.jl#91]
I've discovered things break if I try to interface with mercurial through Julia's shell commands (using ; in the REPL). I don't know enough to know what's going on behind the scenes, but somehow invoking external python software in the shell mode is being influenced by PyCall's behavior

Example:

```
shell> hg identify
7fddeb3f5258+ tip

julia> import PyCall

julia> PyCall.@pyimport sys

shell> hg identify
ImportError: No module named site
```

I'm using the most recent version of Julia, 0.4.0-dev+719 (2014-09-21)

**Comment by stevengj** [comment]
It looks like `Py_GetPath` is called by `Py_InitializeEx` when Python is initialized, so it should be sufficient to only set `PYTHONHOME` while calling this function.  I'll fix this as part of #167.

**Comment by phobon** [comment]
Thanks for the followup!
